tests/test_fonticon/test_collapsible.py

```diff
@@ -1,9 +1,32 @@
 """A test module for testing collapsible"""
+from pathlib import Path
 
+import pytest
 from qtpy.QtCore import QEasingCurve
 from qtpy.QtWidgets import QPushButton
 
 from superqt import QCollapsible
+from superqt.fonticon import icon
+from superqt.fonticon._qfont_icon import QFontIconStore
+
+TEST_PREFIX = "ico"
+TEST_CHARNAME = "smiley"
+TEST_CHAR = "\ue900"
+TEST_GLYPHKEY = f"{TEST_PREFIX}.{TEST_CHARNAME}"
+FONT_FILE = Path(__file__).parent / "icontest.ttf"
```

Comment:
please don't duplicate all this code from `test_fonticon.py`.  If you need these fixtures here as well, place them somewhere that they can be used in both files